packages/core/admin/admin/src/components/LanguageProvider/tests/LocaleStorageMock.js

```diff
@@ -0,0 +1,23 @@
+class LocalStorageMock {
```

Comment:
So glad you've put this one here 👍🏻

Reply:
Yeah I am thinking that I should put it in the jest `setupTestFrameworkScriptFile` config.